inclusive-design/idrc
# chore: update build process to align with starter kit

## Description

Updates build process to use the one included in [fluidic-11ty](https://github.com/fluid-project/fluidic-11ty).
## Steps to test

1. Run in dev mode: `npm run start`
2. Run a production build: `npm run build`

**Expected behavior:** Builds are successful.

## Additional information

Not applicable.

## Related issues

Not applicable.
